BuildToolsCliVersion.txt

```diff
@@ -0,0 +1 @@
+1.0.0-preview2-002733
```

Comment:
We should be using stage0 instead of another CLI version.  We only need to download 1 CLI to build.